app/controllers/index.rb

```diff
@@ -0,0 +1,63 @@
+get '/session-viewer' do
+  session.inspect
+end
+
+get '/session-reset' do
+  session[:user_password] = nil
+  session[:user_id] = nil
+  session.inspect
```

Comment:
Routes that make changes should never be a `get`. Use `post` for non-readonly actions.